client/queryResultsContentProvider.ts

```diff
@@ -79,7 +79,11 @@ export class QueryResultsContentProvider implements TextDocumentContentProvider
         } else {
             window.showInformationMessage(`Query in ${uri.query} got an empty response from ${uri.authority}`)
         }
-        const responseUri = Uri.parse(`${QueryResultsContentProvider.scheme}://${uri.authority}${uri.path}.${fmt}?${uri.query}`)
+        let validUriPath = uri.path
+        if (validUriPath.startsWith('//')) {
```

Comment:
Nice - do you know how the URI path was ending up with "//" in the path?

Reply:
There's another function that builds the URI initially. It is adding a "/" because sometimes the next value doesn't have one. I probably could have changed the URI building there. I'm not sure there's a difference either way.